runtime/plugins/org.jboss.tools.runtime.ui/src/org/jboss/tools/runtime/ui/internal/dialogs/FastProgressMonitorFocusJobDialog.java

```diff
@@ -148,32 +149,15 @@ public void clearBlocked() {
 	}
 	
 	private void addRunableMonitorAsListener(Job longJob, IProgressMonitorWithBlocking listener) {
-		IProgressMonitorWithBlocking jobMonitor = ProgressManager.getInstance().progressFor(longJob);
+		JobMonitor jobMonitor = ProgressManager.getInstance().progressFor(longJob);
```

Comment:
@jeffmaury : refactored since builds fails with:
```04:37:41 [ERROR] /mnt/hudson_workspace/workspace/jbosstools-base_master/sources/runtime/plugins/org.jboss.tools.runtime.ui/src/org/jboss/tools/runtime/ui/internal/dialogs/FastProgressMonitorFocusJobDialog.java:[151] 
04:37:41 [ERROR] 	IProgressMonitorWithBlocking jobMonitor = ProgressManager.getInstance().progressFor(longJob);
04:37:41 [ERROR] 	                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
04:37:41 [ERROR] Type mismatch: cannot convert from ProgressManager.JobMonitor to IProgressMonitorWithBlocking```